src/vs/workbench/contrib/chat/common/chatModel.ts

```diff
@@ -1556,8 +1563,9 @@ export class ChatModel extends Disposable implements IChatModel {
 		return request;
 	}
 
-	setCustomTitle(title: string): void {
+	public setCustomTitle(title: string): void {
 		this._customTitle = title;
+		this._onDidChange.fire({ kind: 'setCustomTitle', title });
```

Comment:
Do you need this event?

Reply:
I don't see the update until I swap tabs if I do not fire the event